types/setting.go

```diff
@@ -1103,6 +1109,26 @@ var (
 		ReadOnly:    true,
 		Default:     "1024",
 	}
+
+	SettingDefinitionTolerateEmptyNodeSelectorVolume = SettingDefinition{
+		DisplayName: "Tolerate Empty Node Selector Volume",
```

Comment:
NIT: How about renaming this to `Allow scheduling empty selector volumes to any nodes`? The first glance of `Tolerate` mislead me that it is about the Kubernetes taint & toleration feature LOL

Reply:
Yes, I got this naming from the `taint & toleration`
but your are right it might confuse users

Reply:
@ChanYiLin Also need to change the setting to `SettingAllowEmptyNodeSelectorVolume`. Please re-check the places needed to update.